sp/News/service1/src/main/java/edu/hillel/service1/config/RestTemplateConfig.java

```diff
@@ -0,0 +1,22 @@
+package edu.hillel.service1.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.client.RestTemplate;
+
+@Configuration
+public class RestTemplateConfig {
+
+    @Value("${news.api.key}")
+    private String apiKey;
+
+    @Bean
+    public RestTemplate restTemplate() {
```

Comment:
What for did you use RestTemplate in this service?

Reply:
You're right. In this case basic RestTemplate instance is being created without any additional configuration. I forgot to erase. It's because a day before I was trying to work using proxy-server and it looked like this:
    @Bean
    public RestTemplate restTemplate() {
        SimpleClientHttpRequestFactory requestFactory = new SimpleClientHttpRequestFactory();

        Proxy proxy = new Proxy(Proxy.Type.HTTP, new InetSocketAddress(proxyHost, proxyPort));
        requestFactory.setProxy(proxy);

        if (proxyUsername != null && proxyPassword != null) {
            Authenticator.setDefault(new Authenticator() {
                @Override
                protected PasswordAuthentication getPasswordAuthentication() {
                    return new PasswordAuthentication(proxyUsername, proxyPassword.toCharArray());
                }
            });
        }
        return new RestTemplate(requestFactory);
    }